backend/src/gamestate/game.rs

```diff
@@ -289,17 +326,8 @@ impl Game {
         };
         let drawn_cards = self.draw_n_cards(player_name.clone(), draw_count);
 
-        self.end_turn();
-        self.message_all_but(
-            player_name.clone(),
-            WSMsg::draw(
-                player_name,
-                self.get_current_player().unwrap().name(),
-                draw_count,
-            ),
-        );
-
-        Ok(drawn_cards)
+        assert_eq!(draw_count, drawn_cards.len());
```

Comment:
Toto patří do testů, tady nám to může potenciálně shodit server

Reply:
@mazanec202 A my to na serveru běžíme v debug módu, ne v releasu?

Reply:
Obávám se, že Rustu je to jedno...

> Assertions are always checked in both debug and release builds, and cannot be disabled.

 https://doc.rust-lang.org/std/macro.assert.html

`debug_assert_eq!` bude to, co hledáme

Reply:
Fixed in rebase.